examples/text_classification/iterable_train.py

```diff
@@ -25,15 +49,30 @@ def generate_batch(batch):
     return text, offsets, label
 
 
+r"""
+torch.utils.data.DataLoader is recommended for PyTorch users to load data.
+We use DataLoader here to load datasets and send it to the train()
+and text() functions.
+"""
+
+
 def train(lr_, num_epoch, data_):
+    r"""
+    Here we use SGD optimizer to train the model.
+
+    Arguments:
+        lr_: learning rate
+        num_epoch: the number of epoches for training the model
+        data_: the data used to train the model
+    """
     data = DataLoader(
         data_,
         batch_size=batch_size,
         collate_fn=generate_batch,
         num_workers=args.num_workers,
         pin_memory=True)
     optimizer = torch.optim.SGD(model.parameters(), lr=lr_)
-    scheduler = torch.optim.lr_scheduler.ExponentialLR(optimizer, args.lr_gamma)
+    scheduler = torch.optim.lr_scheduler.StepLR(optimizer, 1, gamma=args.lr_gamma)
```

Comment:
are the default argument names set to sane values for this?

Also, could you add some more docs to the arguments for these scripts?

In particular you use the [help keyword](https://docs.python.org/3/library/argparse.html#adding-arguments) for this. You should then also include the default in that help strings.